internal/report/report.go

```diff
@@ -1211,10 +1226,10 @@ func NewDefault(prof *profile.Profile, options Options) *Report {
 	return New(prof, o)
 }
 
-// computeTotal computes the sum of all sample values. This will be
-// used to compute percentages.
+// computeTotal computes the sum of the absolute value of all sample values.
```

Comment:
Doc should be updated when such non-trivial logic is added in the function code.

Reply:
done.